hack/make.sh

```diff
@@ -39,13 +39,12 @@ DEFAULT_BUNDLES=(
 )
 
 VERSION=${VERSION:-dev}
-if [[ $VERSION == refs/tags/* ]]; then
-	VERSION=${VERSION#refs/tags/}
-elif [[ $VERSION == refs/heads/* ]]; then
-	VERSION=$(sed <<< "${VERSION#refs/heads/}" -r 's#/+#-#g')
-elif [[ $VERSION == refs/pull/* ]]; then
-	VERSION=pr-$(grep <<< "$VERSION" -o '[0-9]\+')
-fi
+case "$VERSION" in
+	refs/tags/v*) VERSION=${VERSION#refs/tags/v} ;;
+	refs/tags/*) VERSION=${VERSION#refs/tags/} ;;
```

Comment:
curious; do we need _both_ ? i.e., wouldn't stripping the `v` just be a no-op if none is present?

Reply:
We don't need the non-v case, because currently we only run the `bin-image` for only for tags which start with `v`:

https://github.com/moby/moby/blob/c3c1ee01505f4071f817304ea0d5cde7bf359dc8/.github/workflows/bin-image.yml#L13-L14

But in case we'd have that in the future or/and the script is used in some other context, we'd still want to transform `refs/tags/whatever` into `whatever`.

Reply:
Oh! 🤦 I was reading it wrong, I thought the substitution was just stripping the `v` prefix (`${VERSION#v}`), but it's replacing `refs/tags/v`... ignore me, ignore me.